### PR TITLE
ADD orderBy for id and type

### PR DESCRIPTION
**Pull request by fgalan**
Partial implementation of #2934

### CODE REVIEW

File: test/functionalTest/cases/2934_order_by_id_and_type/order_by_id_and_type_combined.test [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+--SHELL--
+
+#
+# 01. Create entity E1/A, name: Daniel
+# 02. Create entity E2/A, name: Carl
+# 03. Create entity E1/B, name: Bob
+# 04. Create entity E2/B, name: Alice
+# 05. Get sorted by id, then name: E1/B/Bob, E1/A/Daniel, E2/B/Alice, E2/A/Carl
+# 06. Get sorted by id, then reserve name: E1/A/Daniel, E2/B/Bob, E2/A/Carl, E2/A/Alice
```
Comment (kzangeli) [inline test/functionalTest/cases/2934_order_by_id_and_type/order_by_id_and_type_combined.test]: I was wondering ... "reserve name" ... what can that be ...
There is nothing more ...
Then I understood! :-)
Typo: reserve => reverse (many places)

File: test/functionalTest/cases/2934_order_by_id_and_type/order_by_id_and_type_combined.test [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+  "type": "B",
+  "name": "Alice"
+}'
+orionCurl --url /v2/entities?options=keyValues -X POST --payload "$payload"
+echo
+echo
+
+
+echo "05. 05. Get sorted by id, then name: E1/B/Bob, E1/A/Daniel, E2/B/Alice, E2/A/Carl"
+echo "================================================================================="
```
Comment (kzangeli) [inline test/functionalTest/cases/2934_order_by_id_and_type/order_by_id_and_type_combined.test]: 05. 05.

File: test/functionalTest/cases/2934_order_by_id_and_type/order_by_id_and_type_combined.test (hunk 1 above)
Comment (fgalan) [reply test/functionalTest/cases/2934_order_by_id_and_type/order_by_id_and_type_combined.test]: Fixed in 1b4d8c65132814aa96ccf2c4f45249a5e4e9f32e

File: test/functionalTest/cases/2934_order_by_id_and_type/order_by_id_and_type_combined.test (hunk 2 above)
Comment (fgalan) [reply test/functionalTest/cases/2934_order_by_id_and_type/order_by_id_and_type_combined.test]: Fixed in 1b4d8c65132814aa96ccf2c4f45249a5e4e9f32e

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**